### PR TITLE
98 Fixed linting issues due to unused variable or properties as found…

### PR DESCRIPTION
**Pull request by Yashvardhan-17**
… by detekt

### Contributor checklist

-   [x] This pull request is on a [98-fix-unuesed-variable-lint](https://github.com/Yashvardhan-17/Scribe-Android/tree/feature/98-fix-unuesed-variable-lint) and not the main branch

---

### Description

This pull request fixes unused variable warnings by removing them as per reported by Detekt. These changes improve code quality and maintainability. I ran the application and verified that there are no warnings related to unused variables in the codebase after my changes.

-   #ISSUE_NUMBER
-  #98

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks so much for the PR, @Yashvardhan-17! 😊

Looks like there are a few errors via ktlint and detekt: https://github.com/scribe-org/Scribe-Android/actions/runs/10935017413/job/30359947247. Do you want to check those out and send along fixes? :)

**Review by angrezichatterbox — COMMENTED** [review]
Thanks for your quick PR @Yashvardhan-17. Some changes to be made and it would be good for merging :)

File: build.gradle.kts [hunk 1]
```diff
@@ -11,7 +11,7 @@ buildscript {
     }
 
     dependencies {
-        classpath("com.android.tools.build:gradle:8.6.0")
+        classpath("com.android.tools.build:gradle:8.2.2")
```
[the remark this answers is not in the document]
Comment (Yashvardhan-17) [reply build.gradle.kts:14]: yes i will.

**Comment by Yashvardhan-17** [comment]
Hey, I have made the mentioned changes.

**Comment by andrewtavis** [comment]
I dropped Gradle back to v8.6.0 as switching between Gradle versions has proven to be a bit annoying on my machine, and I'd prefer to just jump on minor versions rather than patches (unless there's a need). Hope this makes sense, or maybe we can also discuss how to update Gradle locally in the call tomorrow, @angrezichatterbox! At the very least we can get ot 8.6.1, but I just didn't want to go through a separate process to bring this in :)

**Review by andrewtavis — APPROVED** [review]
Thanks so much for the PR, @Yashvardhan-17! Is really great to see so much of these legacy variables from the original project we forked from coming out of the project. Hope you pick another issue up soon! :)

**Comment by Yashvardhan-17** [comment]
Thanks @andrewtavis, I will pick another issue up soon :).

**Pull request merged**